public/docs/ts/latest/guide/architecture.jade

```diff
@@ -386,7 +370,7 @@ figure
 
   **Structural** directives alter layout by adding, removing, and replacing elements in DOM.
 
-  Our [example](#templates) template uses two built-in structural directives:
+  The [example](#templates) template uses two built-in structural directives:
```

Comment:
Here and in similar cases, I'd change the link from [example] to [example template]. (since you're linking to the template source, and not the source for the entire example)